<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegCashMain</class>
 <widget class="QMainWindow" name="RegCashMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1219</width>
    <height>716</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1219</width>
    <height>716</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1219</width>
    <height>716</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RegCashMain</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>701</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Codice Articolo</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtArticolo">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>251</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="maxLength">
      <number>9</number>
     </property>
     <property name="placeholderText">
      <string>Codice Articolo</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtDescArt">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>421</width>
       <height>151</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="placeholderText">
      <string>Decrizione</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>701</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="customTable" name="tbCassa">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>681</width>
       <height>291</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="columnCount">
      <number>5</number>
     </property>
     <column>
      <property name="text">
       <string>Articolo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Descrizione</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Sconto</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignVCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Prezzo Vendita</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignVCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Prezzo Scontato</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>-10</y>
      <width>261</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Totale</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtTotale">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 139, 139);</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Compensazione</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>140</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblComp">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>54</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>230</y>
      <width>261</width>
      <height>341</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Sconto Fisso</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtSconto">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="inputMask">
      <string>00%</string>
     </property>
     <property name="text">
      <string>0%</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>....</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Scontrini</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>000000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btnContanti">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>61</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="regcashresource.qrc">
       <normaloff>:/immagini/cash.png</normaloff>:/immagini/cash.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAtm">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>61</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="regcashresource.qrc">
       <normaloff>:/immagini/atm.png</normaloff>:/immagini/atm.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>130</y>
       <width>61</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="regcashresource.qrc">
       <normaloff>:/immagini/gift.png</normaloff>:/immagini/gift.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>-10</y>
      <width>221</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Sconto 10 %</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Sconto 15 %</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Sconto 20 %</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Sconto 30 %</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Sconto 40 %</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>460</y>
      <width>231</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Logo</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1219</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCassa"/>
   <addaction name="actionSconti"/>
   <addaction name="actionArticoli"/>
   <addaction name="actionExportData"/>
   <addaction name="actionSyncShop"/>
   <addaction name="actionImpostazioni"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCassa">
   <property name="icon">
    <iconset>
     <normaloff>../../../Scaricati/cash_register.png</normaloff>../../../Scaricati/cash_register.png</iconset>
   </property>
   <property name="text">
    <string>Cassa</string>
   </property>
  </action>
  <action name="actionSconti">
   <property name="icon">
    <iconset resource="regcashresource.qrc">
     <normaloff>:/immagini/discount.png</normaloff>:/immagini/discount.png</iconset>
   </property>
   <property name="text">
    <string>Sconti</string>
   </property>
  </action>
  <action name="actionArticoli">
   <property name="icon">
    <iconset resource="regcashresource.qrc">
     <normaloff>:/immagini/item.png</normaloff>:/immagini/item.png</iconset>
   </property>
   <property name="text">
    <string>Articoli</string>
   </property>
  </action>
  <action name="actionExportData">
   <property name="icon">
    <iconset resource="regcashresource.qrc">
     <normaloff>:/immagini/trends.png</normaloff>:/immagini/trends.png</iconset>
   </property>
   <property name="text">
    <string>Esportazione Dati</string>
   </property>
  </action>
  <action name="actionSyncShop">
   <property name="icon">
    <iconset resource="regcashresource.qrc">
     <normaloff>:/immagini/radio.png</normaloff>:/immagini/radio.png</iconset>
   </property>
   <property name="text">
    <string>Sincronizza Shop</string>
   </property>
  </action>
  <action name="actionImpostazioni">
   <property name="icon">
    <iconset resource="regcashresource.qrc">
     <normaloff>:/immagini/tools.png</normaloff>:/immagini/tools.png</iconset>
   </property>
   <property name="text">
    <string>Impostazioni</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>customTable</class>
   <extends>QTableWidget</extends>
   <header>customtable.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="regcashresource.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>qqq()</slot>
 </slots>
</ui>
